CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+cmake_minimum_required(VERSION 3.6)
+project(Pokopomclion)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_BUILD_TYPE Debug)
```

Comment:
As per this article https://stackoverflow.com/questions/19024259/how-to-change-the-build-type-to-release-mode-in-cmake, it looks like for Visual Studio you have to specify it when you actually run cmake, like this:
```cmake --build ../ --target ALL_BUILD --config Release```
I can provide these instructions somewhere if you'd like, or include a batch file. Also, building with the original VS solution file rather than cmake still works if users prefer that.

I really appreciate your help reviewing these changes!